app/models/daos/UserDAOImpl.scala

```diff
@@ -91,7 +91,7 @@ object UserDAOImpl {
    * Total number of contributing users
    */
   def countContributingUsers: Int = db.withTransaction { implicit session =>
-    val count = size - 1 + countAnonymousUsers
+    val count = size - 1 + countAnonymousUsers + countTurkerUsers
```

Comment:
Since turkers actually have entries in the user table, they are being double counted here. They are currently being included in the counts of registered users. So what I think you should do is to
1. Modify the countRegisteredUsers function to subtract the number of turkers from it, so it then counts the number of registered _volunteers_
2. Instead of using size - 1, call the countRegisteredUsers function (which, really, should have been done before anyway)

Reply:
@maddalihanumateja has informed me that I am wrong on this, turker do not have a User role, so point 1. can be disregarded. Point 2 still stands.